scripts/ci.release.js

```diff
@@ -29,10 +29,10 @@ function validateEnv() {
 		return false;
 	}
```

Comment:
What is this? Doesn't seem right... Can you also get that fixed? It's either the statement is wrong, or the log message is.